ShowMyData/DrugCrimeSchema.json

```diff
@@ -0,0 +1,74 @@
+﻿{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "type": "array",
+  "items": {
+    "$ref": "#/definitions/Element"
+  },
+  "definitions": {
+    "Element": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "address_x": {
+          "type": "string"
```

Comment:
As long as you're adding a schema, you might as well add validation rules to the data itself as well.  How long can a string be?  What's the maximum and minimum values for a number?